authentik/stages/authenticator_validate/stage.py

```diff
@@ -143,6 +144,8 @@ def validate(self, attrs: dict):
         self.stage.executor.plan.context[PLAN_CONTEXT_METHOD_ARGS]["mfa_devices"].append(
             self.device
         )
+        self.device.last_used = now()
+        self.device.save()
```

Comment:
should be wrapped in `with audit_ignore():` so this doesn't create extra event log entries